USAGE.md

```diff
@@ -187,8 +187,8 @@ aws lambda invoke \
   ```console
   $ aws lambda invoke \
      --function-name "${ENV}-import-status" \
-     --payload '{"httpMethod": "GET", "body": {"execution_arn": "arn:aws:batch:ap-southeast-2:xxxx:job/example-arn"}}' \
+     --payload '{"http_method": "GET", "body": {"execution_arn": "arn:aws:batch:ap-southeast-2:xxxx:job/example-arn"}}' \
      /dev/stdout
 
-  {"statusCode": 200, "body": {"validation":{ "status": "SUCCEEDED"}, "metadata upload":{"status": "Pending", "errors":[]}, "asset upload":{"status": "Pending", "errors":[]}}}
+  {"status_code": 200, "body": {"validation":{ "status": "SUCCEEDED"}, "metadata upload":{"status": "Pending", "errors":[]}, "asset upload":{"status": "Pending", "errors":[]}}}
```

Comment:
`metadata upload` should probably be `metadata_upload`, and `asset upload` should be `asset_upload`.

Reply:
Thank you @l0b0 . I'll fix that.

Reply:
Should be fixed in 68ac4de5d1bc28b22b92d10ba9169953060f3e7b